mava/systems/tf/maddpg/training.py

```diff
@@ -1312,16 +1312,14 @@ def _forward(self, inputs: reverb.ReplaySample) -> None:
 
                 # Critic loss.
                 critic_loss = recurrent_n_step_critic_loss(
-                    q_values,
-                    rewards[agent],
-                    discount * agent_discount,
-                    target_q_values,
+                    q_values=q_values,
```

Comment:
Not sure if this is where some of the bugs got in, but I guess in the future we should always prefer keyword arguments as input instead of positional ones. 😄

Reply:
Agreed :)